scripts/logfile-parser.sh

```diff
@@ -6,11 +6,11 @@ LogParser() {
     while IFS= read -r line; do
         echo "$line"
 
-        if [[ "$line" == *"is using new name"* ]]; then
+        if [[ "$line" == *"] player "* ]]; then
```

Comment:
I think we should use something more specific in here, maybe change it to
```sh
if [[ "$line" == "[userid:"*"] player"*"connected islocalplayer="* ]]; then
```

Reply:
Thanks for the review, I updated the code 👌